frontend/src/services/gameApiService.js

```diff
@@ -0,0 +1,19 @@
+// Service for making request to game API.
```

Comment:
I like this design. I think we should refactor all of the rest of our websocket communcation to go through services like this.

Reply:
So like a chatApiService? or a drawApiService?

Reply:
I havnt really investigated too much into how to actually use those services, right now they are a bunch of singletons, which I think is ok if we dont care about the state of these services.

Reply:
I just think its nicer to not have the webSocketService being used all over the place, and have a "layer" of components that the vue components use instead

Reply:
Ok i can agree with that, I think ideally we want to move logic away from components